caf-audit-binding-elasticsearch/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditConnection.java

```diff
@@ -29,9 +29,12 @@
 
     public ElasticAuditConnection() throws AuditConfigurationException
     {
-            final String hostAndPorts = 
+            final String hostValues = 
                 System.getProperty(ElasticAuditConstants.ConfigEnvVar.CAF_ELASTIC_HOST_AND_PORT_VALUES,
-                                   System.getenv(ElasticAuditConstants.ConfigEnvVar.CAF_ELASTIC_HOST_AND_PORT_VALUES));
+                                   System.getenv(ElasticAuditConstants.ConfigEnvVar.CAF_ELASTIC_HOST_VALUES));
```

Comment:
I don't understand what you're trying to do here.  You are potentially putting the value of `CAF_ELASTIC_HOST_AND_PORT_VALUES` into `hostValues`.

Reply:
If the variable CAF_ELASTIC_HOST_AND_PORT_VALUES has the value, it should be like **elasticsearch:9200**.
 If not the value from CAF_ELASTIC_HOST_VALUES will be **elasticsearch**. Only one value will be available at any time.  Looks like the variable name looks irrelevant.
Because CAF_ELASTIC_HOST_AND_PORT_VALUES variable is now removed from Verity/deploy, always we get the value of CAF_ELASTIC_HOST_VALUES only. So I used variable name hostValues.

Reply:
Too messy - you shouldn't have a variable `hostValues` variable that could contain something other than just host names.

Reply:
Still too messy - now you have a `hostAndPortValues` variable that may not contain ports.

Reply:
Now the code is refactored to hostAndPort variable, to hold the value with hostName and port.